.travis.yml

```diff
@@ -2,7 +2,6 @@ language: php
 
 php:
   - 5.3
-  - 5.4
```

Comment:
Nothing, I wanted to reduce Travis queues as we test almost every version of PHP (5.4 is EOT while 5.5 should cover tests for it without problems).